normandy/control/static/control/js/components/DeleteRecipe.jsx

```diff
@@ -14,8 +14,12 @@ class DeleteRecipe extends React.Component {
             <div className="form-action-buttons">
               <div className="fluid-2 float-right">
                 <input type="submit" value="Confirm" class="delete" onClick={(e) => {
-                  dispatch(ControlActions.makeApiRequest('deleteRecipe', { recipeId }));
-                  dispatch(push(`/control/`));
+                  e.preventDefault();
```

Comment:
This is sufficiently complex that it should probably be a method on the component rather than embedding it here.